packages/tests/server/react/useQueries.test.tsx

```diff
@@ -95,3 +95,32 @@ test('different queries', async () => {
     expect(utils.container).toHaveTextContent(`bar`);
   });
 });
+
+test('mapping queries', async () => {
+  const ids = ['1', '2', '3'];
+
+  const { proxy, App } = ctx;
+  function MyComponent() {
+    const results = proxy.useQueries((t) =>
+      ids.map((id) => t.post.byId({ id })),
+    );
+
+    results[0]?.data;
```

Comment:
Can you test the type here using `expectTypeOf`

Reply:
done